### ISSUE TITLE
[UX] Translation page: Improve comment scope usage

### ISSUE THREAD

**Issue by rvanlaak** [WeblateOrg/weblate#1060]
Quick UX improvements to the translation page (see #1057):
- What does the comment scope do? It only shows the gray label at the right of the comment? Do not think people understand that, we think it should always be 'translation comment' right? 
- Would make this a simple toggle next to the 'save' button
- or maybe remove it completely?
##

**Comment by nijel** [comment]
It defines where the comment is shown. If for source string, then it's shown in all translations, if it's for translation, it's shown only in given translation. This is to allow discuss separate things:

- source string feedback or improvement suggestions
- translation among translators into given language

I'm really open on how to make these two work intuitively without disturbing everybody with both kind of comments :-).

**Comment by rvanlaak** [comment]
From our `monolingual` point of view we would never want to make a difference between comments on a language. Basically comments written for an English translation also are important for the French translation. So we would always prefer the 'all translations' scope. 

In that case adding a simple on/off toggle with the label "make locale specific" would clarify it's behavior right? You have a use case where you actually only want to show it with one translation?

**Comment by paulpach** [comment]
Then how about changing the description:

"Show in all translations"
"Show only in French translation"

The current text is very confusing,  I had no idea what it did either.

**Comment by dg0yt** [comment]
What I observe is that translators use the comments for discussion, often using the default source scope when the translation scope would be correct (e.g. for discussion in the target language). That's why I created #1259. Instead of the scope field after the comment field, the scope should be more explicit. IMO Options include:
- Change, or don't provide, the default value for scope.
- Move the scope field to before the comment field.
- Move the comments to collapsible elements in the Translate frame. For example:
  - Indicate "[NUMBER] comments" in the Source and Translation headlines.
  - Using these elements for (un-)collapsing the comments, which would be displayed next to the field they refer to: source or translation.

In addition I'm not satisfied with the visibility of the comments I provide from the source code (#933). But I think this is a separate issue, indeed, because comments from the source code should always be visible.

BTW, suggestions might need another scope. I saw some suggestions which I would have liked to commented on (usually wrong tense), but the link would have not been visible. (I wasn't able to give the correct translation.)
